plugin.audio.kxmxpxtx.bandcamp/default.py

```diff
@@ -140,11 +146,19 @@ def main():
             query = xbmcgui.Dialog().input(addon.getLocalizedString(30103))
         if query:
             search(query)
+    elif mode[0] == 'url':
+        url = args.get("url", None)[0]
+        build_song_list(*bandcamp.get_album_by_url(url))
+        xbmc.executeJSONRPC(
```

Comment:
Could you explain why this is needed?

Reply:
Yatse devs requested API in order to support share from Bandcamp Android app to Kodi:
https://community.yatse.tv/t/send-to-kodi-from-bandcamp/2389
Calls are needed in order to autoplay first song from the playlist. This is the best way I came up with.
Ideally I need:
1. Make first item selected (tried with .select(True), it kinda worked, but looks like focus and selected item are different things in Kodi)
2. Get list item based on selected item
3. Extract url from list item
4. Start streaming
 I tried different things in order to set first item in focus and play it without json-rpc, but nothing worked for me.

Reply:
Have you tried creating a xbmc.Playlist object, add all songs to the playlist and set the resolved url to the first item on the playlist?

Reply:
Hey, it worked. Removed json-rpc hack. Add the commit to the PR. Thanks!